index.ts

```diff
@@ -0,0 +1,21 @@
+export * from './src'
+
+type EnvVars = Partial<{
+    RPC_AMOY: string,
+    CONTRACT_REGISTRY_AMOY: string,
+    RPC_CARDONA: string,
+    CONTRACT_REGISTRY_CARDONA: string,
+    TEST_KEY_1: string
+    TEST_KEY_2: string
+    // TEST_WITH_CHANGE_OWNER: string|number|boolean
```

Comment:
What is this?

Reply:
It is used for typescript to recognize the environment variables available in process.env, these variables allow us to customize the rpcs that will be used for testing, replace hard-coded public rpcs with "private" rpcs and make tests faster many times and thus avoid the limitations of public rpcs.

Also add private keys to test transactions in the contract, such as changing the owner of the did ethr, etc.

Reply:
I understand, but my doubt was more about why these variables are mentioned and what they do.

Reply:
I pushed a file (`types.d.ts`) that has comments explaining some things about ENV VARS